js/src/client.ts

```diff
@@ -1358,34 +1361,91 @@ export class Client implements LangSmithTracingClientInterface {
     );
   }
 
+  private async _createNodeFetchBody(parts: MultipartPart[], boundary: string) {
```

Comment:
question: is it `node-fetch` or node's native fetch (`undici`?)

Reply:
I think both?